_opensearch/install/quickstart.md

```diff
@@ -0,0 +1,169 @@
+
+    # Using wget:
+    wget https://github.com/opensearch-project/documentation-website/blob/{{site.opensearch_version}}/assets/examples/ecommerce-field_mappings.json
+    ```
+1. Download [ecommerce.json](https://github.com/opensearch-project/documentation-website/blob/{{site.opensearch_version}}/assets/examples/ecommerce.json). This file contains the index data formatted so that it can be ingested by the bulk API. To learn more, see [index data]({{site.url}}{{site.baseurl}}/opensearch/index-data/) and [Bulk]({{site.url}}{{site.baseurl}}/api-reference/document-apis/bulk/).
+    ```bash
+    # Using cURL:
+    curl -O https://github.com/opensearch-project/documentation-website/blob/{{site.opensearch_version}}/assets/examples/ecommerce.json
+
+    # Using wget:
+    wget https://github.com/opensearch-project/documentation-website/blob/{{site.opensearch_version}}/assets/examples/ecommerce.json
+    ```
+1. Define the field mappings with the mapping file.
+    ```bash
+    curl -H "Content-Type: application/x-ndjson" -X PUT "https://localhost:9200/ecommerce" -ku admin:admin --data-binary "@ecommerce-field_mappings.json"
+    ```
+1. Upload the index to the bulk API.
+    ```bash
+    curl -H "Content-Type: application/x-ndjson" -X PUT "https://localhost:9200/ecommerce/_bulk" -ku admin:admin --data-binary "@ecommerce.json"
+    ```
+1. Query the data using the search API. The following command submits a query that will return documents where `customer_first_name` is `Sonya`.
+    ```bash
+    curl -H 'Content-Type: application/json' -X GET "https://localhost:9200/ecommerce/_search?pretty=true" -ku admin:admin -d' {"query":{"match":{"customer_first_name":"Sonya"}}}'
+    ```
+1. Access OpenSearch Dashboards by opening `http://localhost:5601/` in a web browser on the same host that is running your OpenSearch cluster.
+    - The default username is `admin` and the default password is `admin`.
+1. Select the **menu** icon in the top left, then select **Dev Tools** under **Management**.
+1. In the left pane of the console, enter the following:
+    ```json
+    GET ecommerce/_search
+    {
+        "query": {
+            "match": {
+                "customer_first_name": "Sonya"
+            }
+        }
+    }
+    ```
+1. Select the triangle icon at the top right of the request to submit the query. You can also submit the request by pressing `Ctrl+Enter` (or `Cmd+Enter` for Mac users). To learn more about using the OpenSearch Dashboards console for submitting queries, see [Running queries in the console]({{site.url}}{{site.baseurl}}/dashboards/run-queries/).
+
+Add the query parameter `pretty=true` to OpenSearch API requests that return a JSON to see a more readable version of the response body. Otherwise, the response will be a flat JSON. For more information about `pretty` and other query parameters, see [Common REST parameters]({{site.url}}{{site.baseurl}}/opensearch/common-parameters/).
```

Comment:
Maybe specify here that this is only relevant for the `curl` command? In DevTools it's pretty by default.

Reply:
Let me see if I can make that an extra bullet point further up, rather than a "tip" box at the end.